charts/linkerd2-multicluster-remote-setup/templates/gateway.yaml

```diff
@@ -0,0 +1,95 @@
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: {{ .Values.gatewayNamespace }}
```

Comment:
I created this all in a separate namespace because it was kinda on the side at the time. Should it all just go into the control plane namespace? I could see downsides either way.